app/pages/home-for-user/news-pullout.jsx

```diff
@@ -101,6 +101,10 @@ const NewsSection = React.createClass({
   render() {
     const projLink = `${window.location.origin}/projects/${this.state.newestProject.slug}`;
     const avatarSrc = !!this.state.newestAvatar ? this.state.newestAvatar.src : null;
+    const recentProjects = 
+      this.props.updatedProjects
+      .sort((a, b) => { return new Date(b.updated_at) - new Date(a.updated_at); })
```

Comment:
Oh wait a minute, but then I notice the code makes a call to projects later with `getProjectsForPreferences`, so if the sorting is needed here to sort the projects you eventually get, that makes sense.